foundations/installations/command_line_basics.md

```diff
@@ -95,16 +95,16 @@ This section contains helpful links to related content. It isn’t required, so
 * [Video Series from LearnLinuxTv](https://www.youtube.com/playlist?list=PLT98CRl2KxKHaKA9-4_I38sLzK134p4GJ) contains 24 videos explaining the basics of the command line. Videos are brief enough for beginners but, at the same time, detailed enough to get you started and light your inner curiosity.
 
 ### Knowledge Check
-This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
-
-* What is the command line?
-* How do you open the command line on your computer?
-* How can you navigate to a particular directory?
-* Where will `cd` on its own navigate you to?
-* Where will `cd ..` navigate you to?
-* How do you display the name of the directory you are currently in?
-* How do you display the contents of the directory you are currently in?
-* How do you create a new directory?
-* How do you create a new file?
-* How do you destroy a directory or file?
+This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+
+* <a class="knowledge-check-link" href="#command-line">What is the command line?</a>
+* <a class="knowledge-check-link" href="#open-command-line">How do you open the command line on your computer?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-cd">How can you navigate to a particular directory?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid31">Where will `cd` on its own navigate you to?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid30">Where will `cd ..` navigate you to?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-pwd">How do you display the name of the directory you are currently in?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-ls">How do you display the contents of the directory you are currently in?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid7">How do you create a new directory?</a>
+* <a class="knowledge-check-link" href="#create-file">How do you create a new file?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid9">How do you destroy a directory or file?</a>
 * How do you rename a directory or file?
```

Comment:
For this, I would link to the [mv command](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10) from the assignment link.

Reply:
> For this, I would link to the [mv command](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10) from the assignment link.

Thanks for helping me out! I've added that link now.